packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts

```diff
@@ -71,7 +88,7 @@ function resolveCurrentVisitingIdentityInner(
     const avatarSelector = searchAvatarSelector()
     const avatarMetaSelector = searchAvatarMetaSelector()
     const assign = async () => {
-        await delay(500)
+        await delay(5000)
```

Comment:
hmmm why?

Reply:
Twitter is too slow to update the avatar, so you need to wait a little longer

Reply:
Ok... Please add a comment to this line